source/carbonstorage.rst

```diff
@@ -87,17 +87,10 @@ This section outlines the specific data used by the model. See the Appendix for
    * **c_soil**: Carbon density in soil [units: megagrams/hectare]
    * **c_dead**: Carbon density in dead matter [units: megagrams/hectare]
 
- *Example:* Hypothetical study with five LULC classes. Class 1 (Forest) contains the most carbon in all pools. In this example, carbon stored in above- and below-ground biomass differs strongly among land use classes, but carbon stored in soil varies less dramatically.
-
-  ====== ================== ======= ======= ====== ======
-  lucode LULC_name          C_above C_below C_soil C_dead
-  ====== ================== ======= ======= ====== ======
-  1      Forest              140     70      35     12
-  2      Coffee              65      40      25     6
-  3      Pasture/grass       15      35      30     4
-  4      Shrub/undergrowth   30      30      30     13
-  5      Open/urban          5       5       15     2
-  ====== ================== ======= ======= ====== ======
+  .. csv-table:: **Example Carbon Pools Table**
+       :file: ../invest-sample-data/Carbon/carbon_pools_willamette_modified.csv
+       :header-rows: 1
+       :widths: auto
```

Comment:
I wonder if we can still include the table description here... not that it really is providing much context you couldn't get from looking at the table values.

Reply:
Like the carbon edge table, this one is labeled `Table 5`. I guess it's just incrementing on the whole doc...

Reply:
I put the description back and changed it to make sense with the data